apps/web/src/components/docs/VotingWidget.tsx

```diff
@@ -0,0 +1,38 @@
+import { ActionButton, IconThumbDownAlt, IconThumbUpAlt } from '@novu/design-system';
+import { css } from '../../styled-system/css';
+import { Flex, styled } from '../../styled-system/jsx';
+import { text } from '../../styled-system/recipes';
+
+const Text = styled('p', text);
+
+export type Voting = 'up' | 'down';
+
+export const VotingWidget = ({
+  voted,
+  onVoteClick,
+}: {
+  voted: Voting | undefined;
+  onVoteClick: (vote: Voting) => () => void;
+}) => {
+  return (
+    <Flex
+      className={css({
+        marginTop: '250',
+      })}
+      align="center"
+      gap="125"
+    >
+      <Text>Did you find it useful?</Text>
+      <Flex gap="50" align="center">
+        <ActionButton
+          onClick={onVoteClick('up')}
+          Icon={() => <IconThumbUpAlt color={voted === 'up' ? 'white' : undefined} size={20} />}
+        />
+        <ActionButton
+          onClick={onVoteClick('down')}
+          Icon={() => <IconThumbDownAlt color={voted === 'down' ? 'white' : undefined} size={20} />}
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Use `cva` here please‏